pkg/controller/cainjector/setup.go

```diff
@@ -95,13 +96,14 @@ func RegisterAllInjectors(ctx context.Context, mgr ctrl.Manager, namespace strin
 	// Registers a c/r controller for each of APIService, CustomResourceDefinition, Mutating/ValidatingWebhookConfiguration
 	// TODO: add a flag to allow users to configure which of these controllers should be registered
 	for _, setup := range injectorSetups {
-		log := ctrl.Log.WithName(setup.objType.GetName())
-		log.Info("Registering new controller")
+		log := ctrl.Log.WithValues("kind", setup.resourceName)
+		log.Info("Registering a reconciler for injectable")
```

Comment:
Thanks.

> I0131 11:17:08.916286   14868 setup.go:100] cert-manager "msg"="Registering a reconciler for injectable" "kind"="mutatingwebhookconfiguration"
I0131 11:17:08.916577   14868 setup.go:100] cert-manager "msg"="Registering a reconciler for injectable" "kind"="validatingwebhookconfiguration"
I0131 11:17:08.916762   14868 setup.go:100] cert-manager "msg"="Registering a reconciler for injectable" "kind"="apiservice"
I0131 11:17:08.917038   14868 setup.go:100] cert-manager "msg"="Registering a reconciler for injectable" "kind"="customresourcedefinition"